client/components/ShareButton.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2020 Draw authors.
```

Comment:
Let's remove this for now, no other file has a license header.

Reply:
Done.